.github/workflows/continuous_production.yml

```diff
@@ -0,0 +1,47 @@
+name: Continuous production
+on:
+  - push
+  # - pull_request # already covered by push
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Go 1.13
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.13
+        id: go
+
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v2
+
+      - name: Get dependencies
+        run: |
+          go get -v -t -d ./...
+          if [ -f Gopkg.toml ]; then
+              curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
+              dep ensure
+          fi
+      - name: Build
+        run: go build -v .
+      - name: Run tests
+        run: go test ./...
+  deploy:
+    name: Serverless deploy master branch
+    runs-on: ubuntu-latest
+    needs: build
+    # Uncomment after the build is green on this PR
+    # if: github.ref == 'refs/heads/master'
+    steps:
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v2
+      - name: serverless deploy
```

Comment:
I think we should 

1) Make sure the AWS command is installed

2) make sure the AWS credentials are setup properly through something like
```bash
echo "[dont-slack-evil-hackaton]" >> ~/.aws/credentials
echo "aws_access_key_id=$AWS_ACCESS_KEY_ID" >> ~/.aws/credentials
echo "aws_secret_access_key=$AWS_SECRET_ACCESS_KEY" >> ~/.aws/credentials
```
(disclaimer: I did not try the commands above)

3) run the deploy through the makefile:
```
- name Serverless deploy
  run: make deploy
```

Reply:
@gjgd shoot, the serverless deploy command is ran from a docker container, so basically setting the AWS credentials on the main deployment machine of github actions had no effect =_= cc https://github.com/gjgd/dont-slack-evil/pull/9/checks?check_run_id=464617154

Reply:
> Or we can just not use their provided deploy-action and install our own aws cli + serverless npm package

I think that's the best approach

Reply:
https://github.com/gjgd/dont-slack-evil/pull/11

Reply:
All of this still in a Docker container ?